src/Tribe/Models/Post_Types/Organizer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Models an Organizer.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Models\Post_Types
+ */
+
+namespace Tribe\Events\Models\Post_Types;
+
+use Tribe\Models\Post_Types\Base;
+
+/**
+ * Class Organizer
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Models\Post_Types
+ */
+class Organizer extends Base {
+
+	/**
+	 * {@inheritDoc}
+	 *
+	 * @since TBD
+	 */
+	protected function get_cache_slug() {
+		return 'organizers';
+	}
+
+	/**
+	 * {@inheritDoc}
```

Comment:
:no_entry_sign: Doc comment for parameter "$filter" missing

<sub>hash: c3c1df14766413edd69abad18e1c7968</sub>